modules/search/instant-search/components/overlay.scss

```diff
@@ -4,16 +4,17 @@
 	width: 100vw;
 	height: 100vh;
 	opacity: 0.975;
-	transition: opacity 0.5s linear, 0.25s transform ease-in-out;
+	transition: opacity 0.5s linear, 0.5s transform ease-in-out;
```

Comment:
If we end up changing the amount the [overlay has to move](https://github.com/Automattic/jetpack/pull/14320/files#r366365031), we can revert this back to `0.25s` and have essentially the same speed.